cbbr_build/sql/assign_geoms.sql

```diff
@@ -0,0 +1,29 @@
+-- Assign geoms based on geocoding returned values
+ALTER TABLE cbbr_submissions
+ADD geom GEOMETRY;
+
+--clean up empty values
+UPDATE cbbr_submissions
```

Comment:
This update is not needed if cbbr_submission is loaded to recipes with 
`EMPTY_STRING_AS_NULL=YES`
Check other recipes for reference
#2